### DISCUSSION TITLE
ui可以单独二次开发么？

### DISCUSSION THREAD

**Discussion by wangjieCode** [apolloconfig/apollo#4397] [Q&A] [answered]
我们项目中配置了大量的长json， 但是默认的json编辑器只是一个文本框，缺少了很多的功能，有方案可以弥补么

**Comment by nobodyiam** [comment, marked as answer]
Currently, the editor is very simple, so you are welcome to enhance the editor :-)

**Reply by wangjieCode** [reply]
Are there any guidelines for secondary development?

**Reply by nobodyiam** [reply]
There is a general [CONTRIBUTING](https://github.com/apolloconfig/apollo/blob/master/CONTRIBUTING.md) guide. You may also refer the [development guide](https://www.apolloconfig.com/#/zh/development/apollo-development-guide).
